server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/view/cassandra/configuration/EventsourcingConfigurationManagementTest.java

```diff
@@ -85,12 +92,40 @@ void storeShouldThrowWhenConfigurationIsNull(EventStore eventStore) {
     @Test
     void storeShouldThrowWhenBucketCountIsLessThanTheCurrentOne(EventStore eventStore) {
         EventsourcingConfigurationManagement testee = createConfigurationManagement(eventStore);
-        testee.store(SECOND_CONFIGURATION);
+        testee.store(CassandraMailQueueViewConfiguration.builder()
+            .bucketCount(DEFAULT_BUCKET_COUNT)
+            .updateBrowseStartPace(DEFAULT_UPDATE_PACE)
+            .sliceWindow(ONE_HOUR)
+            .build());
 
-        assertThatThrownBy(() -> testee.store(FIRST_CONFIGURATION))
+        assertThatThrownBy(() -> testee.store(CassandraMailQueueViewConfiguration.builder()
+                .bucketCount(DEFAULT_BUCKET_COUNT - 1)
+                .updateBrowseStartPace(DEFAULT_UPDATE_PACE)
+                .sliceWindow(ONE_HOUR)
+                .build()))
             .isInstanceOf(IllegalArgumentException.class);
     }
 
+    @Test
+    void storeShouldWorkWhenIncreaseBucketCount(EventStore eventStore) {
```

Comment:
please don't use "work" in test title, try to find better words. Here I think that "apply" or "accept" are better names